python-sdk/python/eppo_client/_eppo_client.pyi

```diff
@@ -6,7 +6,23 @@ def init(config: ClientConfig) -> EppoClient: ...
 def get_instance() -> EppoClient: ...
 
 class Configuration:
-    def __init__(self, flags_configuration: bytes) -> None: ...
+    # Note: Before publshing v5.0.0
+    #
+    # Users current have to initialize the configuration like this:
+    #
+    # Configuration(some_bytes, some_other_bytes)
+    #
+    # In v5.0.0, we will add a named parameter to the constructor to make this
+    # more clear.
+    #
+    # Users will be able to initialize the configuration like this:
+    # 
+    # Configuration(flags_configuration=some_bytes, bandits_configuration=some_other_bytes)
+    #
+    # Changes:
+    # 
+    # def __init__(self, *, flags_configuration: bytes, bandits_configuration: bytes | None = None) -> None: ...
```

Comment:
It actually already has this signature, so no need to wait for v5:
https://github.com/Eppo-exp/rust-sdk/blob/427fed8fd54f3bdc7b69a9f2e198a72ef5ba4779/python-sdk/src/configuration.rs#L22